components/Buttons/button.css

```diff
@@ -0,0 +1,205 @@
+/* @import url('../../components/root.css'); */
+
+
+@import url('../../components/framer.css');
+
+
+.btn-py-1{
+    padding:0.7rem 2rem 0.7rem 2rem;
+
+}
```

Comment:
If padding is same for top, bottom and right, left you can use shorthand 
padding: 0.7rem 2rem 
It will give the same effect.

Reply:
+1